pglast/__init__.py

```diff
@@ -127,4 +127,4 @@ def _remove_stmt_len_and_location(parse_tree):
 
 
 __all__ = ('Error', 'Missing', 'Node', 'enums', 'get_postgresql_version',
-           'parse_plpgsql', 'parse_sql', 'prettify', 'split')
+           'parse_plpgsql', 'parse_sql', 'prettify', 'split', 'fingerprint')
```

Comment:
Nitpicking hat: can you please keep the symbols in alphabetic order here and in the import statements?

Reply:
Fixed